staffs/views.py

```
@@ -1,3 +1,377 @@
+            return self.get_paginated_response(serializer.data)
+        serializer = TeacherWorkloadSerializer(queryset, many=True, context={'request': request})
+        return Response(serializer.data)
+    
+class DepartmentViewSet(viewsets.ModelViewSet):
+    """
+    Handles Department administration and Staff-to-Department assignments.
+    """
+    queryset = Department.objects.all().order_by('-id')
+    serializer_class = DepartmentSerializer
+    pagination_class = DepartmentPagination
+    filter_backends = [DjangoFilterBackend, SearchFilter, OrderingFilter]
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)